docs/pages/academy/5-reasoner/2-inference-rules.md

```diff
@@ -11,15 +11,17 @@ toc: false
 KB: academy
 ---
 
-As anticipated in the [last lesson](./reasoner-intro.html), in this module you are going to learn how to use Grakn Reasoner to make your data more intelligent.
+As anticipated in the [last lesson](./reasoner-intro.html), in this module you are going to learn how to use the Grakn Reasoner to make your data more intelligent.
 
-Among others, the two most common uses for Reasoner are:
-Knowledge discovery: as in the deserted island example you have read about in the last lesson, we can use logic inference to extract more knowledge out of our data without modifying it.
-Query shortening: sometimes you use the same long query over and over again. It is convenient in those cases to use reasoner to simply create new "virtual" relationship that will make running those same queries in a more succint way.
-In this and the following lesson, we will build an example that will demonstrate both these points.
+Among others, the two most common uses for reasoners are:
+
+  1. Knowledge discovery: as in the deserted island example you have read about in the last lesson, we can use logic inference to extract more knowledge out of our data without modifying it.
+  1. Query shortening: sometimes you use the same long query over and over again. It is convenient in those cases to use reasoners to simply create new "virtual" relationships that make those same queries more succint.
+
+In this and the following lesson, we will build an example that will demonstrate both use cases.
```

Comment:
I'll treat it as a big change then and put it in the respective pull request. Both points are numbered 1. because markdown processors usually take the collection of items and enumerate them in the transpilation process. There are a couple of other documents in the academy using this style (only one in which the items are enumerated "by hand" - which I didn't change because I saw no reason to do so).